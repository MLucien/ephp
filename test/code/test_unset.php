<?

$users = array ( 
	array (
		"name" => "Manuel",
		"surname" => "Rubio",
	), array (
		"name" => "Margarita",
		"surname" => "Ortiz",
	)
);

unset($users[1]);

?>
<html>
<head>
    <title>My Web Example</title>
</head>
<body>
    <pre><?=print_r($users,true)?></pre>
</body>
</html>

