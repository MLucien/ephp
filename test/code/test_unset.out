<html>
<head>
    <title>My Web Example</title>
</head>
<body>
    <pre>Array
(
    [0] => Array
    (
            [name] => Manuel
            [surname] => Rubio
    )
)
</pre>
</body>
</html>

